src/foam/dao/CachingDAO.js

```diff
@@ -87,6 +93,15 @@ foam.CLASS({
           promise: cacheFilled
         });
       }
+    },
+    {
+      class: 'Int',
+      name: 'pollingFrequency'
```

Comment:
frequency is times/unit-of-time, like 3hz, 10 times / minute, etc. but this is a duration/delay/interval, so maybe pollingInterval would be better. And add:
units: 'ms',